valhalla/skadi/sample.h

```diff
@@ -61,6 +81,21 @@ class sample {
 
   cache_t* cache_;
   friend cache_t;
+
+  /**
+   * Get a single sample from cache
+   * @param coord the single posting at which to sample the datasource
+   */
+  template <class coord_t> double get_from_cache(const coord_t& coord);
+
+private:
+  std::string url_;
+  std::unique_ptr<baldr::tile_getter_t> remote_loader_;
+  std::unordered_set<std::string> st_;
+  std::mutex st_lck_;
+  std::uint32_t num_threads_{1};
```

Comment:
I guess this is an abandoned line the previous commits. Could drop it?

Reply:
it is still used in [sample](https://github.com/valhalla/valhalla/blob/5eb2fdd718103452f516b729f0d913be50050d12/src/skadi/sample.cc#L369) construction

Reply:
I mean `num_threads_` is not used anywhere beside `constructor`